wave_matching/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
+PROJECT(wave_matching)
+
+IF (CMAKE_COMPILER_IS_GNUCXX)
+    ADD_DEFINITIONS(-std=gnu++11)
+ENDIF()
```

Comment:
this is defined in the master branch in global `CMakeFile.txt` so this is not needed.